controller/workitem_blackbox_test.go

```diff
@@ -1676,8 +1677,8 @@ func (s *WorkItem2Suite) TestWI2UpdateWithArea() {
 		},
 	}
 	_, wi := test.CreateWorkitemCreated(t, s.svc.Context, s.svc, s.wi2Ctrl, c.Data.Relationships.Space.Data.ID.String(), &c)
-	assert.NotNil(t, wi.Data.Relationships.Area)
-	assert.Nil(t, wi.Data.Relationships.Area.Data)
+	require.NotNil(t, wi.Data.Relationships.Area)
+	assert.NotNil(t, wi.Data.Relationships.Area.Data.ID)
```

Comment:
can we add a verification that will check if `area.data.id == rootArea of that space`